packages/css/src/list/list.scss

```diff
@@ -1,16 +0,0 @@
-/**
```

Comment:
We want to keep a seperate List component, right? So maybe we should keep this file, the React component and the story

Reply:
Sure, perhaps we should deal with it story DES-23